src/ui/common/Main/MainContainer.tsx

```diff
@@ -0,0 +1,13 @@
+import { useAppSelector } from "../../../app/hooks";
+import { RootState } from "../../../app/store";
+import MainComponent from "./MainComponent"
+
+const MainContainer = () =>
+{
+    const isOpened = useAppSelector((state: RootState) => state.sidebar.open);
+    return (
+        <MainComponent isOpened = {isOpened}/>
```

Comment:
formatting should be <MainComponent isOpened={isOpened} />
Please, add the Prettier - Code formatted to your VS Code

Reply:
Formatting has been improved using Prettier